src/script/script_error.h

```diff
@@ -8,7 +8,7 @@
 
 #include <string>
 
-typedef enum ScriptError_t
+using ScriptError = enum ScriptError_t
```

Comment:
In commit "refactor: use "using" over "typedef"" (dfd911500bdd1a9542f79d1a823b1b66245ad76c)

Would be good for a followup (possible future PR) to drop `ScriptError_t` in tests and just change this to `enum ScriptError`